simplq/src/components/pages/Home/StaticInfos.jsx

```diff
@@ -35,18 +35,23 @@ export const HowToCreate = () => (
         <ul className={styles['list-info']}>
           <img src="/images/wand.svg" alt="Wand" />
           <li>
-            <h3>Create Queue</h3>
-            <p>Enter queue name of your choice and create a brand new queue totally free of cost</p>
+            <h3>Create a Queue</h3>
+            <p>Enter a queue name and click on the create queue button to generate a new queue</p>
           </li>
```

Comment:
Since we have manage your queue later on, how about using create your queue and join your queue instead?